gatsby-config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
     {
       resolve: 'gatsby-source-mathdroid-covid19',
       options: {
-        countries: [{ iso2: 'DE' }],
+        countries: [{ iso2: process.env.COVID_COUNTRY }],
```

Comment:
Should the country live in the .env or should it live within gatsby config? Especially given that the component that uses the country data is expecting an array of countries

Reply:
Great catch, I think for now we can limit this to one country and later on we can have comma separated param